src/components/chipFormatter/ChipFormatter.jsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState, useEffect } from 'react';
+import { Chip } from '@mui/material';
+
+const ChipFormatter = ({ value }) => {
+  const [color, setColor] = useState();
+
+  useEffect(() => {
+    switch (value) {
```

Comment:
First of all you can move switch statement to searate function. Second, why are you doing this check in useEffect

Reply:
done